ui/app/app.js

```diff
@@ -112,7 +112,7 @@ export default class App extends Application {
         services: ['flash-messages', 'router', 'store', 'version'],
         externalRoutes: {
           kvSecretDetails: 'vault.cluster.secrets.backend.kv.secret.details',
-          clientCountOverview: 'vault.cluster.clients.overview',
+          clientCountOverview: 'vault.cluster.clients',
```

Comment:
Rather than having to change this again if the target route changes I updated to the top level route which has a redirect so we will only need to update in one place in the future.